core/src/main/java/edu/wpi/grip/core/exception/GripException.java

```diff
@@ -0,0 +1,26 @@
+
+package edu.wpi.grip.core.exception;
+
+/**
+ * An exception thrown when something goes wrong with an internal GRIP
+ * operation.
+ */
+public class GripException extends RuntimeException {
+
+    public GripException() {
```

Comment:
I'm not a huge fan of having this exception in the first place. But I'm having a hard time justifying telling you to remove it as well.

Regardless. Please remove this constructor. A message should be required for this exception.